admiral/common_functions.sqf

```diff
@@ -93,14 +93,13 @@ adm_common_fnc_setGear = {
 adm_common_fnc_tryRemoveNVGs = {
     FUN_ARGS_1(_unit);
 
+    {
+        if (_x in assignedItems _unit) exitWith {
```

Comment:
Small optimization if you store `assignedItems _unit` in a variable, so the loop doesn't have to evaluate it every time. This is just a minor note. I might be wrong and the interpreter is smart enough to figure this out on it's own. Also we know the loop runs on a constant array, so it's not a problem at all.